src/scss/_accordion.scss

```diff
@@ -87,14 +88,28 @@
   line-height: px-to-rem(24);
 
   border-left: px-to-rem(3) solid jazz-theme(accordion-content-border-color);
-  margin: px-to-rem(10) px-to-rem(20);
+  margin: 0 px-to-rem(20);
   padding-left: px-to-rem(20);
-  margin-bottom: px-to-rem(16);
   text-align: left;
+  visibility: hidden;
+  padding-top: 0;
+  padding-bottom: 0;
+  height: 0;
+  opacity: 0;
+  transform: translateY(-25%);
+  transition: .15s ease-in-out;
 }
 .jazz-accordion__content[aria-hidden='true'] {
   display: none;
 }
+.jazz-accordion__content.expanded {
```

Comment:
Can you rename expanded to jazz-accordion_content--expanded? Since expanded is such a common phrase/verb we want to make sure that this does not get overridden by any other classes that are also named expanded.

Reply:
- [ ] change .expanded to jazz-accordion_content--expanded